src/server/config/cmds.go

```diff
@@ -53,6 +53,38 @@ func deduceActiveEnterpriseContext(cfg *config.Config) (string, error) {
 	return activeEnterpriseContext, nil
 }
 
+func ConnectCmds() []*cobra.Command {
+	var commands []*cobra.Command
+
+	connect := &cobra.Command{
+		Short: "Connect to a Pachyderm Cluster",
+		Long:  "Connect to a Pachyderm Cluster",
+		Run: cmdutil.Run(func(args []string) (retErr error) {
+			address := args[0]
+			cfg, err := config.Read(false, false)
+			if err != nil {
+				return err
+			}
+			var context config.Context
+			pachdAddress, err := grpcutil.ParsePachdAddress(address)
+			if err != nil {
+				if !errors.Is(err, grpcutil.ErrNoPachdAddress) {
+					return err
+				}
+			} else {
+				context.PachdAddress = pachdAddress.Qualified()
+			}
+			name := "local"
+			cfg.V2.Contexts[name] = &context
+			cfg.V2.ActiveContext = name
+			return cfg.Write()
```

Comment:
It would be nice if we did a quick sanity check.  My initial thought with this feature would be to run "auth login", but I'm not sure what happens if you "auth login" when auth is off (probably a confusing error message).  Ignoring that, I think it would be nice to do an InspectCluster call just to check that the connection works before writing the file.  Maybe that's annoying though; interested in your thoughts there.

Reply:
I changed things around here a bit and added some more guardrails and output. I also set the context name to the passed in address vs the fully qualified one. It also checks to see if there's a matching context first and if so, doesn't modify it (so it doesn't blow away any auth or other stuff that might be saved there)

Reply:
w.r.t connecting sanity checks, I think I'd like to leave those out for now, I'd like to keep this command "offline" for now, but we could potentially add those checks in a second pass at this command.

Reply:
OK, this looks good to me.  Switching context by address is a really good idea.